src/fireEvent.js

```diff
@@ -5,13 +5,25 @@ import { ErrorWithStack } from './helpers/errors';
 const findEventHandler = (
   element: ReactTestInstance,
   eventName: string,
-  callsite?: any
+  callsite?: any,
+  nearestHostDescendent?: ReactTestInstance
```

Comment:
can we flip the order so `callsite` stays at the end?

Reply:
I've tried it, but the drawback is that in the root call-site it changes

```js
const handler = findEventHandler(element, eventName, callsite);
```

to
```js
const handler = findEventHandler(element, eventName, undefined, callsite);
```

With that undefined argument actually being implementation details for recursive function.

Reply:
👍